File: 프로그래머스/lv2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
import java.util.Arrays;

class Solution {
    public String solution(String s) {
        int[] nums = Arrays.stream(s.split(" "))
                            .mapToInt(Integer::parseInt)
                            .toArray();
        int max = Integer.MIN_VALUE;
        int min = Integer.MAX_VALUE;
        for(int num : nums) {
            if (max < num) max = num;
            if (num < min) min = num;
        }
        StringBuilder answer = new StringBuilder();
        answer.append(min);
        answer.append(" ");
        answer.append(max);
        return answer.toString();
    }
}

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.kt
class Solution {
    fun solution(s: String): String {
        var min = Int.MAX_VALUE
        var max = Int.MIN_VALUE
        for (i in s.split(" ").map{it.toInt()}) {
            if (i > max) {
                max = i
            }
            if (i < min) {
                min = i
            }
        }
        return "$min $max"
    }
}